ConsoleApp/Shop/Buyer.cs

```diff
@@ -0,0 +1,7 @@
+namespace ConsoleApp.Shop;
+using System.Collections.Generic;
+
+class Buyer
+{
+    private string _name { get; set; }
```

Comment:
code convention violated here
```suggestion
    private string Name { get; set; }
```